src/main.rs

```diff
@@ -15,6 +20,32 @@ fn parse_matches() -> ArgMatches<'static> {
                 .default_value("default")
                 .help("namespace to deploy test cluster"),
         )
+        .arg(
+            Arg::with_name("deploy_method")
+                .long("deploy-method")
+                .takes_value(true)
+                .possible_values(&["local", "tar", "skip"])
+                .default_value("local")
+                .help("Deploy method. tar, local, skip. [default: local]"),
+        )
+        .arg(
+            Arg::with_name("local-path")
+                .long("local-path")
+                .takes_value(true)
+                .required_if("deploy-method", "local")
+                .conflicts_with_all(&["tar", "skip"])
```

Comment:
If I'm reading the docs correctly, this should only work if `"tar"` and `"skip"` are their own args, and not variants of "deploy_method". Does this work as expected?